### DISCUSSION TITLE
Drupal 5?

### DISCUSSION THREAD

**Discussion by jscottelblein** [retrofit-drupal/retrofit#138] [Q&A]
I know I read on the main site that it handles D7, but does it also handle D5?

If not, is anyone aware of a similar module (or something) that will?

**Comment by mglaman** [comment]
Right now there are functions from Drupal 7. As far as I know, there aren't any other projects doing this for earlier versions.

Now, there is a chance this project could help run Drupal 5 code. I guess it depends what the code is. I started using Drupal in 2012 or so and only used Drupal 7.
